src/content/docs/en/guides/fonts.mdx

```diff
@@ -60,9 +49,8 @@ Let’s imagine you have a `DistantGalaxy.woff` font file.
 
 The [Fontsource](https://fontsource.org/) project simplifies using Google Fonts and other open source fonts. It provides npm modules you can install for the fonts you want to use.
```

Comment:
Not something added in your PR but why not use this opportunity to fix this and advocate for "open-source" for consistency's sake.
```suggestion
The [Fontsource](https://fontsource.org/) project simplifies using Google Fonts and other open-source fonts. It provides npm modules you can install for the fonts you want to use.
```

Reply:
Thanks Yan! I'll just commit this so we can merge!  :partying_face: